### ISSUE TITLE
Button not functioning

### ISSUE THREAD

**Issue by Semicolon10** [scorelab/LabelLab#393]
**Describe the bug**
Back  button not functioning in labller 

**To Reproduce**
1. Go to the dashboard
2. Click on New project
3. Give name and description for the project
4. Go to project images and add new image.
5. Click edit image and it will direct you to another dashboard
4. Try to click the back button

**Expected behavior**
Either button should be removed or it should direct to the previous page 

**Screenshots**
![Capture](https://user-images.githubusercontent.com/29561593/77465054-177f6f80-6e2e-11ea-887e-19111a0e69e4.PNG)

**Desktop (please complete the following information):**
 - OS: Windows10
 - Browser Chrome
 - Version latest

**Comment by thivatm** [comment]
@Semicolon10 The back button is there for the purpose of navigating through images(In case if you have added multiple images in the project) did you check the button with multiple images added to the project?

**Comment by laksh22** [comment]
Some suggestions:
1. Don't show the __Back__ button if the user is editing the first image in the list of images    
2. Don't show the __Skip__ button if the user is editing the last image in the list of images
3. Rename __Back__ to __Previous__ and __Skip__ to __Next__. Sounds more intuitive

@Semicolon10 You can think about adding these features and making a PR

**Comment by Semicolon10** [comment]
@thivatm  @laksh22  thank you for the suggestions I ll start working on this

**Comment by temanisparsh** [comment]
@laksh22 If this issue is not resolved yet, I would like to work on this.

**Comment by laksh22** [comment]
@temanisparsh You can start if @Semicolon10 isn't working on this anymore

**Comment by temanisparsh** [comment]
@laksh22 I worked on this and it looks like there is no way to be label multiple images at once. Even selecting multiple images in `Project Images` and then clicking `Edit` selects only a single image. In this case, I don't see the point of having these buttons at all. Please correct me if I am wrong.

**Comment by laksh22** [comment]
@temanisparsh Yes, you can only edit one image at a time. Did you select multiple images by using the checkboxes? Those are just for deleting multiple images.

The point of the __Back__ and __Skip__ buttons is to cycle through the images you have added and edit them without having to go back to the `Project Images` in order to edit another image.
